packages/test/test-driver-definitions/package.json

```diff
@@ -78,7 +78,7 @@
     "typescript-formatter": "7.1.0"
   },
   "typeValidation": {
-    "version": "1.3.0",
+    "disabled": true,
```

Comment:
I don't recall that this one was disabled in other branches, just double checking if this is right.

Reply:
good catch, let me check.